.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
     # Custom environment variables for the ARM wheel builder
     - CIBW_BUILD_VERBOSITY=1
     - CIBW_TEST_COMMAND="bash {project}/build_tools/travis/test_wheels.sh"
-    - CIBW_ENVIRONMENT="CPU_COUNT=6
+    - CIBW_ENVIRONMENT="CPU_COUNT=2
```

Comment:
- [CPU_COUNT=2](https://app.travis-ci.com/github/scikit-learn/scikit-learn/builds/240920017): total time 37 min 3 sec
- [CPU_COUNT=3](https://app.travis-ci.com/github/scikit-learn/scikit-learn/builds/240897656): total time 35 min 50 sec
- [CPU_COUNT=4](https://app.travis-ci.com/github/scikit-learn/scikit-learn/builds/240918207): total time 38 minutes 9 seconds

Since these are all around the same time, I think keeping `CPU_COUNT=2` to avoid memory issues is a good compromise.

Reply:
I agree.